Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.6.5-jessie
+
+env DEBIAN_FRONTEND noninteractive
+RUN apt-get update && \
+    apt-get -y install \
+            dbus-x11 \
+            libdbus-1-dev \
+            libespeak1 \
+            libnotify-cil-dev \
+            notification-daemon \
+            sudo \
+            xorg
+
```

Comment:
Remove apt cache after installing depndencies as mentioned in dockerfile best practices (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) 
```
RUN apt-get update && \
    apt-get -y install \
            dbus-x11 \
            libdbus-1-dev \
            libespeak1 \
            libnotify-cil-dev \
            notification-daemon \
            sudo \
            xorg  && \
     rm -rf /var/lib/apt/lists/*
```